### ISSUE TITLE
can the gt_density be saved as .jpg file

### ISSUE THREAD

**Issue by SophieChang66** [svishwa/crowdcount-mcnn#25]
hello, I saved the gt_density data as .jpg file ,then convert it to lmdb, and then train in caffe , but the loss is huge, like as
 Iteration 9000 (5.64897 iter/s, 177.024s/1000 iters), loss = 3.87866e+06

how can i solve the problem, or something i did is wrong?
